<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmImageSwitch</class>
 <widget class="QWidget" name="frmImageSwitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="ImageSwitch" name="imageSwitch3" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="ImageSwitch" name="imageSwitch1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="ImageSwitch" name="imageSwitch2" native="true"/>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageSwitch</class>
   <extends>QWidget</extends>
   <header>imageswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
